policies/slsa/verify-builder.yaml

```diff
@@ -9,8 +9,6 @@ initiatives:
   - client-policy
 
 evidence:
-  filter-by:
```

Comment:
would you want this filter-by: pipeline ?

Reply:
Good point, but in this case there's another issue, cause the rule error shows the same ID (from the SLSA provenance) that we have in the rule input.

Reply:
maybe the ID should default to current.RULE_ID or something like that?

Reply:
The meaning of this field is actually differs between a pipeline and a local environment. Maybe it's better to split them to different rules or at least to different configs, one defaulting to smth like `{{ github.run_id }}` and the other one leave templated.